lib/node_modules/@stdlib/math/base/special/floor10/src/addon.c

```diff
@@ -19,4 +19,4 @@
 #include "stdlib/math/base/special/floor10.h"
 #include "stdlib/math/base/napi/unary.h"
 
-STDLIB_MATH_BASE_NAPI_MODULE_D_D( stdlib_base_floor10 )
+STDLIB_MATH_BASE_NAPI_MODULE_D_D( floor10_func )
```

Comment:
This is not the desired change. See other `addon.c` files in other packages for what is desired.